core-bundle/src/DependencyInjection/Compiler/MakeServicesPublicPass.php

```diff
@@ -22,43 +22,34 @@
  */
 class MakeServicesPublicPass implements CompilerPassInterface
 {
-    private const SERVICES = [
+    private const IDS = [
         'assets.packages',
+        'database_connection',
         'debug.stopwatch',
         'fragment.handler',
         'lexik_maintenance.driver.factory',
         'monolog.logger.contao',
         'security.authentication.trust_resolver',
+        'security.encoder_factory',
         'security.firewall.map',
-        'security.logout_url_generator',
         'security.helper',
-        'uri_signer',
-    ];
-
-    private const ALIASES = [
-        'database_connection',
+        'security.logout_url_generator',
         'swiftmailer.mailer',
-        'security.encoder_factory',
+        'uri_signer',
     ];
 
     public function process(ContainerBuilder $container): void
     {
-        foreach (self::SERVICES as $service) {
-            if (!$container->hasDefinition($service)) {
-                continue;
+        foreach (self::IDS as $id) {
+            if ($container->hasAlias($id)) {
+                $alias = $container->getAlias($id);
+                $alias->setPublic(true);
```

Comment:
```suggestion
                $alias->setPublic(true);
                
                continue;
```

Reply:
That's even worse IMHO. No hard feelings. 😄